examples.go

```diff
@@ -0,0 +1,142 @@
+package main
+
+import (
+	"bytes"
+	"flag"
+	"fmt"
+	"io"
+	"io/ioutil"
+	"log"
+	"os"
+	"path/filepath"
+
+	"github.com/russross/blackfriday"
+	"github.com/xeipuuv/gojsonschema"
+)
+
+var (
+	verbose bool
+	skipped bool
+)
+
+var specsByMediaType = map[string]string{
+	"application/vnd.oci.image.manifest.v1+json":      "schema/image-manifest-schema.json",
+	"application/vnd.oci.image.manifest.list.v1+json": "schema/manifest-list-schema.json",
```

Comment:
On Fri, Apr 22, 2016 at 07:26:53AM -0700, Brandon Philips:
“Why not name the files exactly the same as the mime type?”

Reasons why I like:

  $ echo application/vnd.oci.image.manifest.v1+json |
  >   sed 's|application/vnd.oci.||;s|[+]|.|'
  image.manifest.v1.json

more are:

1. application/stuff is a filename on Windows and a path into a
   subdirectory on POSIX.
2. vnd.oci. is noise in this repo (although that noise isn't a
   terrible problem if you want to keep it).
3. +json is an odd file extension (although again, you can keep it if
   you like).